Taskfile.dist.yaml

```diff
@@ -35,3 +280,427 @@ tasks:
+        minio/mc mb main/trdl-test-project2
+      - |
+        docker run -ti --rm \
+        -e MC_HOST_main=http://{{.s3_access_key}}:{{.s3_secret_key}}@$(docker inspect {{.minio_container}} --format "{{ "{{ .NetworkSettings.IPAddress }}" }}"):{{.minio_port}} \
+        minio/mc policy set public main/trdl-test-project2
+      - |
+        docker run -ti --rm \
+        -e MC_HOST_main=http://{{.s3_access_key}}:{{.s3_secret_key}}@$(docker inspect {{.minio_container}} --format "{{ "{{ .NetworkSettings.IPAddress }}" }}"):{{.minio_port}} \
+        minio/mc mb main/werf
+      - |
+        docker run -ti --rm \
+        -e MC_HOST_main=http://{{.s3_access_key}}:{{.s3_secret_key}}@$(docker inspect {{.minio_container}} --format "{{ "{{ .NetworkSettings.IPAddress }}" }}"):{{.minio_port}} \
+        minio/mc policy set public main/werf
+      - |
+        docker run -ti --rm \
+        -e MC_HOST_main=http://{{.s3_access_key}}:{{.s3_secret_key}}@$(docker inspect {{.minio_container}} --format "{{ "{{ .NetworkSettings.IPAddress }}" }}"):{{.minio_port}} \
+        minio/mc mb main/trdl
+      - |
+        docker run -ti --rm \
+        -e MC_HOST_main=http://{{.s3_access_key}}:{{.s3_secret_key}}@$(docker inspect {{.minio_container}} --format "{{ "{{ .NetworkSettings.IPAddress }}" }}"):{{.minio_port}} \
+        minio/mc policy set public main/trdl
+      - |
+        docker run -ti --rm \
+        -e MC_HOST_main=http://{{.s3_access_key}}:{{.s3_secret_key}}@$(docker inspect {{.minio_container}} --format "{{ "{{ .NetworkSettings.IPAddress }}" }}"):{{.minio_port}} \
+        minio/mc mb main/kubedog
+      - |
+        docker run -ti --rm \
+        -e MC_HOST_main=http://{{.s3_access_key}}:{{.s3_secret_key}}@$(docker inspect {{.minio_container}} --format "{{ "{{ .NetworkSettings.IPAddress }}" }}"):{{.minio_port}} \
+        minio/mc policy set public main/kubedog
+    vars:
+      dir: '{{.dir | default "server"}}'
+
+  setup:vault:
+    desc: Set up and configure Vault
+    dir: "{{.dir}}"
+    cmds:
+      - docker rm -f {{.vault_container}} || true
+      - |
+        docker run --workdir /app --privileged --name {{.vault_container}} \
+        -e VAULT_PLUGIN_SECRETS_TRDL_PPROF_ENABLE=1 \
+        -e VAULT_PLUGIN_SECRETS_TRDL_DEBUG=1 \
+        --detach --volume /var/run/docker.sock:/var/run/docker.sock \
+        --volume $(pwd):/app \
+        -p {{.vault_port}}:{{.vault_port}} \
+        ghcr.io/werf/trdl-dev-vault:latest \
+        server -dev -dev-root-token-id=root \
+        -dev-plugin-dir=/app/vault/plugins -log-level trace
+      - |
+        while ! VAULT_ADDR=http://$(docker inspect {{.vault_container}} --format "{{ "{{ .NetworkSettings.IPAddress }}" }}"):{{.vault_port}} \
+        vault status; do
+          sleep 1
+        done
+      - task: configure:vault
+    vars:
+      dir: '{{.dir | default "server"}}'
+
+  configure:vault:
```

Comment:
Look, how about we move these horrible setup:... and configure:... and vault:logs tasks out of here to some Taskfile.legacy.yaml or something and forget about it.